libcaf_bb/test/stream_reader.cpp

```diff
@@ -0,0 +1,160 @@
+/******************************************************************************
+ *                       ____    _    _____                                   *
+ *                      / ___|  / \  |  ___|    C++                           *
+ *                     | |     / _ \ | |_       Actor                         *
+ *                     | |___ / ___ \|  _|      Framework                     *
+ *                      \____/_/   \_|_|                                      *
+ *                                                                            *
+ * Copyright 2011-2019 Dominik Charousset                                     *
+ *                                                                            *
+ * Distributed under the terms and conditions of the BSD 3-Clause License or  *
+ * (at your option) under the terms and conditions of the Boost Software      *
+ * License 1.0. See accompanying files LICENSE and LICENSE_ALTERNATIVE.       *
+ *                                                                            *
+ * If you did not receive a copy of the license files, see                    *
+ * http://opensource.org/licenses/BSD-3-Clause and                            *
+ * http://www.boost.org/LICENSE_1_0.txt.                                      *
+ ******************************************************************************/
+
+#define CAF_SUITE stream_reader
+
+#include "caf/bb/stream_reader.hpp"
+#include "caf/bb/tokenized_integer_reader.hpp"
+
+#include "caf/test/dsl.hpp"
+
+#include <memory>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "caf/actor_system.hpp"
+#include "caf/actor_system_config.hpp"
+
+using namespace caf;
+
+namespace {
+
+using stream_type = std::istringstream;
+using value_type = bb::tokenized_integer_reader<>::value_type;
+
+TESTEE_SETUP();
+
+TESTEE_STATE(stream_reader_sink) {
+  std::vector<value_type> vec;
+};
+
+TESTEE(stream_reader_sink) {
+  return {[=](stream<value_type> in) {
+    return self->make_sink(
+      // input stream
+      in,
+      // initialize state
+      [=](unit_t&) {
+        // nop
+      },
+      // Consumer
```

Comment:
Please stick to one commenting style.

```suggestion
      // consume values
```